composer.json

```diff
@@ -13,8 +13,9 @@
         "beberlei/doctrineextensions": "^1.5",
         "doctrine/doctrine-migrations-bundle": "^3.3",
         "sentry/sentry-symfony": "^5.0",
-        "sumocoders/framework-core-bundle": "^10.0",
+        "sumocoders/framework-core-bundle": "dev-278-asset-mapper",
```

Comment:
Dat moet dan nog aangepast worden?

Reply:
Ja, net voor te merge (heb het ook in AC gezet als subtask)